### PR TITLE
Con 1261

### PR DESCRIPTION
**Pull request by annapoltieva**
CON-1261

## Problem
Ruby 2.6 is reaching EOL and we need to upgrade.

## Solution
Upgrade tp 3.1.2

### CODE REVIEW

File: lib/api/client.rb [hunk 1]
```diff
@@ -144,7 +144,7 @@ def upload_to_connector(connector_id, filepath, run_now = true, max_retries = 3,
           query_response = RestClient::Request.execute(
             method: :post,
             url: connector_endpoint,
-            headers: headers,
```
Comment (charles-sturgis-kenna) [inline lib/api/client.rb:147]: RestClient's version didn't bump - how do we have a different signature?

Comment (charles-sturgis-kenna) [reply lib/api/client.rb:147]: Are these in response to a rubocop? Also, note the trailing colon - we've got `headers:,` instead of `headers,`

**Review by charles-sturgis-kenna — CHANGES_REQUESTED** [review]
Let's look at these changed signatures

File: lib/api/client.rb (hunk 1 above)
Comment (mukhansk) [reply lib/api/client.rb:147]: Yes rubocop say that now we need write ```headers:,``` instead of ```headers: headers,```

**Review by charles-sturgis-kenna — APPROVED** [review]

**Pull request merged**